src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let res = 0;
+
+  for (let i = 0; i < days; i++) {
```

Comment:
u don't need a loop here)
use mathematic.
If we know the number of days and the price for 1 day - we can know the base price for all vacation, right?